README.rst

```diff
@@ -103,6 +103,18 @@ install xcode by running ``xcode-select --install`` and retry step 2.
 Now everything should be set up. When you want to start scrobbling, simply
 run ``lastcast`` in the terminal.
 
+Docker setup
+----------------------------
+Docker uses "volume" in order to link the configuration file setup into the Docker container.
+On your local machine create a directory containing your configured `lastcast.toml` configuration file
+
+.. code:: bash
+
+   $ configDirectory=YourLocalConfigurationDirectory
+   $ docker pull erik/lastcast
+   $ docker run -it --net=host --name lastcast -v $configDirectory:/config erik/lastcast:lastest
```

Comment:
Been a while since I've written Dockerfiles, does it make sense to add `/config` as a `VOLUME` configuration?

Reply:
I may be misunderstanding but I believe this is using the `VOLUME` configuration via the `-v` flag.

If you were wondering why I did this it was so that I could provide the configured `lastcast.toml` into the known `/config` directory which I am telling lastcast to look in at startup here: https://github.com/erik/lastcast/pull/51/files#diff-3254677a7917c6c01f55212f86c57fbfR14

I may be misunderstanding, thanks!

Reply:
Ah no I was misunderstanding the docs. I was thinking `VOLUME /config` would also be useful to include here, but seems like it wouldn't be.